virt-v2v/cold/entrypoint.go

```diff
@@ -77,7 +77,7 @@ func main() {
 }
 
 func buildCommand() []string {
-	virtV2vArgs := []string{"virt-v2v", "-v", "-x"}
+	virtV2vArgs := []string{"virt-v2v"}
```

Comment:
i think it's important to have `-v -x`. this helps us and users to figure out what is happening or why it fail.

Reply:
yeah, @bkhizgiy - I think you should have posted it as a draft  to emphasis that it's not ready for a comprehensive review (but to share the code changes you had locally)